mechafil/power.py

```diff
@@ -194,15 +200,22 @@ def forecast_power_stats(
             day_qa_renewed_power,
             duration,
         )
-        day_qa_renewed_power[day_i] = compute_day_qa_renewed_power(
-            day_i,
-            day_rb_scheduled_expire_power,
-            renewal_rate_vec,
-            fil_plus_rate,
-            fil_plus_m,
-            duration_m,
-            duration,
-        )
+        
+        # see https://hackmd.io/O6HmAb--SgmxkjLWSpbN_A?view for more details
+        if qap_method == 'tunable':
+            day_qa_renewed_power[day_i] = compute_day_qa_renewed_power(
+                day_i,
+                day_rb_scheduled_expire_power,
+                renewal_rate_vec,
+                fil_plus_rate,
+                fil_plus_m,
+                duration_m,
+                duration,
+            )
+        elif qap_method == 'basic':
+            day_qa_renewed_power[day_i] = compute_day_rb_renewed_power(
```

Comment:
I think we should rename the function `compute_day_rb_renewed_power` to `compute_basic_day_renewed_power` since it can be used now for with RB or QA power.

Reply:
done